tests/driver_hd44780/Makefile

```diff
@@ -0,0 +1,11 @@
+APPLICATION = driver_hd44780
+include ../Makefile.tests_common
+
+BOARD_WHITELIST := arduino-uno
```

Comment:
jep will, do. But currently had only an _uno_ at hand for testing. Basically all boards with 6 GPIOs and above will do. Further, there are i2c drivers for this display, too - I ordered one and will add support soonish.